src/xsct.c

```diff
@@ -223,6 +225,20 @@ int main(int argc, char **argv)
     }
     else
     {
+      /* Check if the temp is above 100 less than the norm and change to NIGHT if it is */
+      /* The threashold was chosen to give some room for varients in temp */
+      if (toggle == 1) {
+        screen = 0;
+        temp = get_sct_for_screen(dpy, screen, crtc_specified, fdebug);
+        if (temp.temp > (TEMPERATURE_NORM - 100)) {
+          temp.temp = TEMPERATURE_NIGHT;
+          sct_for_screen(dpy, screen, crtc_specified, temp, fdebug);
+        }
+        else {
+          temp.temp = TEMPERATURE_NORM;
+          sct_for_screen(dpy, screen, crtc_specified, temp, fdebug);
+        }
+      }
```

Comment:
Thanks!

Requires some style changes for consistency with the code style of this project:
- needs further indentation to match the level below
- should switch to `//` single-line comments as used elsewhere
- `toggle != 0` to match the style of comparisons elsewhere
- curlies are on its own line as elsewhere
- `sct_for_screen` call can be moved out to a call after `temp.temp` update; however, the temperature of all screens should be updated as done elsewhere